test/utils/hstasklisthelper/HsTaskListHelperRegistrationSpec.scala

```diff
@@ -248,8 +246,7 @@ class HsTaskListHelperRegistrationSpec extends AnyWordSpec with Matchers with Mo
       when(mockSpokeCreationService.getAddTrusteeHeaderSpokesToggleOff(any(), any(), any(), any())).thenReturn(testTrusteeEntitySpoke)
       when(mockSpokeCreationService.getAddEstablisherHeaderSpokesToggleOff(any(), any(), any(), any())).thenReturn(testEstablishersEntitySpoke)
 
```

Comment:
val lastUpdatedDate = Instant.now(Clock.fixed(Instant.parse("2022-09-05T00:00:00Z"), ZoneOffset.UTC))
 maybe this line can be added back, so that we know how we are getting that timeStamp Long number?

Reply:
Good point, I've added it back in